docs/docs/05-reusable-components.md

```diff
@@ -232,3 +232,33 @@ Methods follow the same semantics as regular ES6 classes, meaning that they don'
 ### No Mixins
 
 Unfortunately ES6 launched without any mixin support. Therefore, there is no support for mixins when you use React with ES6 classes. Instead, we're working on making it easier to support such use cases without resorting to mixins.
+
+
+
+## Stateless Functions
+
+You may also define your React classes as a plain JavaScript function. For example using the stateless function syntax:
+
+```javascript
+function HelloMessage(props) {
+  return <div>Hello {props.name}</div>;
+}
+React.render(<HelloMessage name="Sebastian" />, mountNode);
+```
+
+Or using the new ES6 arrow syntax:
+
+```javascript
+var HelloMessage = (props) => <div>Hello {props.name}</div>;
+React.render(<HelloMessage name="Sebastian" />, mountNode);
+```
+
+
+This simplified component API is intended for components that are pure functions of their props.  These components must not retain internal state, do not have backing instances, and do not have the component lifecycle methods.  They are pure functional transforms of their input, with zero boilerplate.
```

Comment:
Haha, one space for proportional fonts and two spaces for monospace fonts leaves it ambiguous what to do when you're writing .md which will turn into HTML.  I went ahead and changed this file to single-space, though we aren't exactly consistent throughout the rest of our docs :).